vm/actor/src/builtin/power/mod.rs

```diff
@@ -482,10 +482,15 @@ impl Actor {
             .map_err(|e| {
                 ActorError::new(
                     ExitCode::ErrIllegalArgument,
-                    format!("fault not verified: {}", e.msg()),
+                    format!("fault not verified: {}", e),
+                )
+            })?
+            .ok_or_else(|| {
+                ActorError::new(
+                    ExitCode::ErrIllegalArgument,
+                    "fault not verified".to_owned(),
```

Comment:
Just noting here that we should follow up with the go impls, their implementation does not handle the nil pointer and seems like it would just panic unnecessarily